src/components/utils/api.js

```diff
@@ -2,23 +2,42 @@ import axios from "axios";
 
 const api = axios.create({baseURL: 'https://news-portfolio.onrender.com/api'})
 
-export const getArticles = () => {
-  return api.get('/articles')
+export const getTopics = () => {
+  return api.get('/topics')
     .then(({data}) => {
       return data;
     })
 }
 
+export const getArticles = (topic) => {
+  return api.get(`/articles${topic ? '?topic=' + topic : ''}`)
```

Comment:
I like that you've refactored this function to now take topics as a argument 🙌 
You may find the axios params object useful - it served the same purpose as this template literal and is easy to extend for future tickets 🎫